packages/lib-utils/src/k8s/k8s-resource.ts

```diff
@@ -62,7 +63,7 @@ export const k8sCreateResource = <
   fetchOptions = {},
 }: K8sResourceUpdateOptions<TResource>): Promise<TCreatedResource> =>
   commonFetchJSON.post<TCreatedResource>(
-    getK8sResourceURL(model, resource, queryOptions),
+    getK8sResourceURL(model, resource, omit(queryOptions, 'queryParams')),
```

Comment:
What is the URL out the other end -- because wouldn't the `resource` still make use of the name?

Reply:
Yeah. I jumped the gun on opening a PR. I am working on addressing the name and workspace path params.